packages/components/src/components/chat-search/ContextStatus.vue

```diff
@@ -0,0 +1,197 @@
+<template>
+  <v-status-bar v-if="appmapStats && appmapStats.numAppMaps" level="success">
+    <template v-slot:header>
+      <div class="context__header">{{ appmapStats.numAppMaps }} AppMaps available</div>
```

Comment:
Nitpick - handle the non-plural form
```suggestion
      <div class="context__header">
        {{ appmapStats.numAppMaps }} AppMap{{ appmapStats.numAppMaps === 1 ? '' : 's' }} available
      </div>
```